Source/Core/Core/Movie.cpp

```diff
@@ -818,13 +827,13 @@ void RecordInput(const GCPadStatus* PadStatus, int controllerID)
 }
 
 // NOTE: CPU Thread
-void CheckWiimoteStatus(int wiimote, const u8* data, const WiimoteEmu::ReportFeatures& rptf,
-                        int ext, const wiimote_key key)
+void CheckWiimoteStatus(int wiimote, const DataReportBuilder& rpt, int ext,
+                        const EncryptionKey& key)
 {
-  SetWiiInputDisplayString(wiimote, data, rptf, ext, key);
+  SetWiiInputDisplayString(wiimote, rpt, ext, key);
 
   if (IsRecordingInput())
-    RecordWiimote(wiimote, data, rptf.size);
+    RecordWiimote(wiimote, rpt.GetDataPtr(), rpt.GetDataSize());
```

Comment:
... this should really be recording the report ID as well... otherwise it's going to be impossible to meaningfully edit in an external program (We Need A New DTM File Format)

Reply:
Yes. There are talks of a new file format.